index.html

```diff
@@ -38,91 +38,102 @@
       </ul>
       <span class="close-button" aria-label="Close menu" tabindex="0">✕</span>
     </div>
+    <div class="desktop-navigation__bar" aria-label="Navigation bar">
+      <ul class="desktop-navigation_ul" aria-label="Only desktop navigation options">
+        <li><a class="desktop-ul__li" href="./index.html" tabindex="0">Home</a></li>
+        <li><a class="desktop-ul__li" href="./about.html" tabindex="0">About</a></li>
+      </ul>
+    </div>
```

Comment:
- [ ] **You have made a great design replicating the original design ✨✨** However, they are some missing page links on the navbar, I understand it would not function now, but it's a good practice to follow the design of a template, even if some of the links are not functioning. This allows you to get a better sense of the overall structure and flow of the template 👍 

**Your design** | **Original Design**
| --- | --- |
![image](https://user-images.githubusercontent.com/102757126/225826641-9abcfa8a-aec8-44ed-a0f7-8edfdd2421d9.png) | ![image](https://user-images.githubusercontent.com/102757126/225826812-1d6f7c36-a625-4b23-8040-bc871d202322.png)
![image](https://user-images.githubusercontent.com/102757126/225827151-b703e6d0-3b26-4980-b261-4477871a0da4.png) |

Reply:
Thank you very much. I fixed it.
Have a good day.